geemap/common.py

```diff
@@ -539,7 +570,7 @@ def ee_export_image_collection_to_drive(
     formatOptions=None,
     **kwargs,
 ):
-    """Creates a batch task to export an ImageCollection as raster images to Google Drive.
+    """Creates a batch task to export an ImageCollection to Google Drive.
```

Comment:
I can understand the "as raster images" because it won't be an ImageCollection object in Google Drive. But "raster images" is not quite right either. Just registering a minor amount of concern here. I don't think a change is need right now. (same for following similar changes)